projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.ts

```diff
@@ -53,6 +61,26 @@ export class CartItemComponent implements OnInit {
     );
   }
 
+  ngOnChanges(changes: SimpleChanges) {
+    this.populateCartItemContext(changes);
+  }
+
+  private populateCartItemContext(changes: SimpleChanges) {
+    if (this.cartItemContext) {
+      const newChunk = Object.entries(changes).reduce(
+        (acc, [key, change]) => ({ ...acc, [key]: change.currentValue }),
+        {} as CartItemContextModel
+      );
+
+      let oldChunk: CartItemContextModel;
```

Comment:
The only alternative I found would be using `BehaviorSubject.value`, which is kind of the same(!) in its evil. Moreover I wanted to avoid BehaviorSubject to avoid initial fake value.

Reply:
IMO there is nothing wrong with using BehaviorSubject.value, as long as you can assume, that you are dealing with behavior subject in the first place. 
I also understand, that you want to avoid the initial value (or emission), on the other hand, the whole `subscribe().unsubsrcibe()` trick means, that you actually would benefit from it, as you want to do something even when there is not yet any context available...?

Reply:
Ok, let's go with BehaviorSubject.